e2e/test/scenarios/embedding-sdk/create-question.cy.spec.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  entityPickerModal,
+  modal,
+  restore,
+  visitFullAppEmbeddingUrl,
+} from "e2e/support/helpers";
+import {
+  EMBEDDING_SDK_STORY_HOST,
+  describeSDK,
+} from "e2e/support/helpers/e2e-embedding-sdk-helpers";
+import { JWT_SHARED_SECRET } from "e2e/support/helpers/e2e-jwt-helpers";
+import {
+  getSdkRoot,
+  signInAsAdminAndEnableEmbeddingSdk,
+} from "e2e/test/scenarios/embedding-sdk/helpers/interactive-question-e2e-helpers";
+
+describeSDK("scenarios > embedding-sdk > create-question", () => {
+  beforeEach(() => {
+    restore();
+    signInAsAdminAndEnableEmbeddingSdk();
+
+    cy.signOut();
+  });
+
+  it("can create questions via the CreateQuestion component", () => {
+    cy.intercept("POST", "/api/card").as("createCard");
+
+    visitFullAppEmbeddingUrl({
+      url: EMBEDDING_SDK_STORY_HOST,
+      qs: { id: "embeddingsdk-createquestion--default", viewMode: "story" },
+      onBeforeLoad: (window: any) => {
+        window.JWT_SHARED_SECRET = JWT_SHARED_SECRET;
+        window.METABASE_INSTANCE_URL = Cypress.config().baseUrl;
+      },
+    });
+
+    // Wait until the entity picker modal is visible
+    getSdkRoot().contains("Pick your starting data");
+
+    entityPickerModal().within(() => {
+      cy.findByText("Tables").click();
+      cy.findByText("Orders").click();
+    });
+
+    getSdkRoot().within(() => {
+      // The question title's header should be "New question" by default.
+      cy.contains("New question");
```

Comment:
I got a feedback from Maz that we should probably show the actual model name or table name instead of "New question", but let's do that in a separate PR.

Reply:
I've made a utility function called `getQuestionTitle` in the `QuestionTitle` component (we can split that out) which gets the question name, adhoc description, or `New question` by default. We could probably extend that to be a `getEntityName` type of thing

Reply:
@oisincoveney Yep, I fiddled with `getQuestionTitle` a little bit yesterday but couldn't find an obvious method in the `Question` object that spits out the model or table name. I'll check with the querying folks